game_runner.rb

```diff
@@ -0,0 +1,23 @@
+class GameRunner
+  BOARD_TOP = " __ __ __\n"
```

Comment:
Some feedback I got on constants is that they tend to cause you to look at things from a perspective of "data," when OO is intended to be more about "messages." Moving this to a method would satisfy the latter. It's a little easier to stub a method when testing too, compared to a constant.

Reply:
Great feedback. I actually did make this a method in my "Apply your design guides to the result" branch. I'm not sure what the refactoring expectation is for this step of the process

Reply:
I think you're right. The process is to open a PR for red, a PR for green, then a PR for the refactor. After step 1 you can start to do those three steps in a single PR.

Reply:
great, let's do that. I'll add commits to the same PR moving forward.